modules/service/src/main/resources/db/migration/V0030__observations.sql

```diff
@@ -150,6 +150,11 @@ create table t_observation (
   c_active_status      e_obs_active_status not null    default 'active',
   c_visualization_time timestamp           null        default null,
 
+  -- planned time summary
+  c_pts_pi        interval not null default '0h',
+  c_pts_uncharged interval not null default '0h',
+  c_pts_execution interval not null default '0h',
```

Comment:
This seems fine for now.  How might it eventually be implemented?  It'll have to be a fairly complicated calculation that we won't be able to directly execute in a stored procedure.  Many different kinds of updates to the observation would have to also update these values.

Reply:
Unclear. We may need to recompute it when we do various updates, or just do it on-demand as a top-level computation.